hazelcast/src/test/java/com/hazelcast/jet/impl/submitjob/validator/SubmitJobParametersValidatorForDirectExecutionTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.impl.submitjob.validator;
+
+import com.hazelcast.jet.JetException;
+import com.hazelcast.jet.impl.SubmitJobParameters;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class SubmitJobParametersValidatorForDirectExecutionTest {
```

Comment:
This tests `SubmitJobParametersValidator`, so the class should be named `*SubmitJobParametersValidatorTest`, otherwise it is hard to navigate between class and its test.

Reply:
Revision : 11ab460f976509617bf9cca53cd5dcc5f424e4d2